pnpm-workspace.yaml

```
@@ -0,0 +1,3 @@
+managePackageManagerVersions: true
+resolutionMode: time-based
+minimumReleaseAge: 1440
```

セキュリティリスクの低減を考えると、1日はあまり効果ないような気もしました！
あのnpm問題とかは対処され始めるまで3日とか4日とかでしたっけ。
まあ休日とか狙ってくると思うし、そういうもんかなと。

1週間くらいでどうでしょ！

```suggestion
minimumReleaseAge: 10080
```

---

1周間にしました。